tsl/test/sql/compression_update_delete.sql

```diff
@@ -511,6 +511,39 @@ DELETE from sample_table WHERE device_id = 3 AND val IS NULL;
 SELECT * FROM sample_table WHERE device_id = 3 AND val IS NULL;
 ROLLBACK;
 
+-- test in equality condition
```

Comment:
These tests don't use any decompression optimizations? Can we say that explicitly here?

Reply:
Added comment.